src/config.rs

```diff
@@ -345,6 +355,16 @@ fn parse_xpub(s: &str) -> Result<(XyzPubKey, RescanSince)> {
     Ok((xpub, rescan))
 }
 
+fn parse_descriptor(s: &str) -> Result<(String, RescanSince)> {
+    let mut parts = s.splitn(2, ':');
+    let descriptor = String::from(parts.next().unwrap());
```

Comment:
The two unwraps here should instead return an `Err`, similarly to how its done in `parse_xpub`.